### ISSUE TITLE
PropertySortOrder and `@media` directive

### ISSUE THREAD

**Issue by hail2u** [brigade/scss-lint#274]
When enabling "PropertySortOrder" (default), child properties of `@media` directive should lint its order, but not linted.

For example:

``` scss
.foo {
  color: #001;
  background-color: #000;

  @media screen {
    color: #003;
    background-color: #002;
  }
}
```

SCSS Lint reports error with this SCSS file only in line 2, not line 6.
